test/Interop/Cxx/foreign-reference/Inputs/module.modulemap

```diff
@@ -18,6 +18,11 @@ module Nullable {
   requires cplusplus
 }
 
+module Inheritance {
+    header "inheritance.h"
```

Comment:
Super minor: let's match the rest of this file in terms of indentation (2 spaces vs 4 spaces).